requirements.txt

```diff
@@ -33,7 +33,7 @@ pluggy==1.5.0
 pre-commit==4.2.0
 pydantic==2.11.0b2
 pydantic-settings==2.8.1
-pydcs @ git+https://github.com/dcs-retribution/pydcs@4a5e3ee63d3a487841c7b88c0a5bf58811567d6c
+pydcs @ git+https://github.com/Druss99/pydcs@3f44568904fb1ca2460eaded8bc892415c27a227
```

Comment:
I guess we need to merge some things to pydcs and point this link back to the correct place 😅